docs/input/docs/ci-systems/github.md

```diff
@@ -57,11 +78,12 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           if-no-files-found: warn
-          name: issues
+          name: ${{ matrix.os }} issues
           path: BuildArtifacts/report.html
 
       - name: Upload Packages
         uses: actions/upload-artifact@v2
+        if: matrix.os == 'windows-latest'
```

Comment:
I think you also can use `runner.os == 'windows` (or is it `github.runner.os`), instead of hardcoding the windows image.

Reply:
`runner.os` it is. Nice. I will update that.

Reply:
@AdmiringWorm I updated the code.